generators/server/templates/src/main/docker/central-server-config/application.yml

```diff
@@ -8,3 +8,8 @@ jhipster:
         authentication:
             jwt:
                 secret: my-secret-token-to-change-in-production
+
+eureka:
+    client:
+        serviceUrl:
+            defaultZone: http://admin:admin@registry:8761/eureka/
```

Comment:
The password is not a constant, "admin" is just a default value, this is the last question from the docker-compose sub-generator